src/components/Landing.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import WeatherIcon from 'react-open-weather-icons';
+import { today, weather } from '../helpers/LandFunction';
+
+class Landing extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    weather(this);
+  }
+
+  render() {
+    const day = today();
+    let kel;
+    const { weatherInfo } = this.state;
+    if (weatherInfo) {
+      kel = (weatherInfo.main.temp - 273.15).toFixed(1);
+    }
+    return (
+      <div className="main-body">
+        <p id="des">WEATHER APP</p>
+        <p id="day">{day}</p>
+        <p id="line1">|</p>
+        <p id="deg">
+          {kel}
+°
```

Comment:
* I suggest that this degree sign and the icon, you put space between them because the overlap each other
* And names of day, temp and icons are in black colors, it is hard to see them well.
![Screenshot from 2020-02-21 16-29-47](https://user-images.githubusercontent.com/56886064/75042726-8ed88100-54c7-11ea-9775-4faec34920d5.png)